packages/esm-service-queues-app/src/hooks/useQueueEntries.ts

```diff
@@ -25,7 +25,7 @@ function getInitialUrl(rep: string, searchCriteria?: QueueEntrySearchCriteria) {
 
   if (searchCriteria) {
     for (let [key, value] of Object.entries(searchCriteria)) {
-      if (value != null) {
+      if (value != null && value !== undefined) {
```

Comment:
undefined is nully

```suggestion
      if (value != null) {
```

Reply:
Ohh okay, I just realized, `undefined == null` is true, `undefined === true` is false. Thanks @brandones !

Reply:
And importantly, undefined == false is false, and null == false is false.